src/v0/destinations/gainsight_px/util.js

```diff
@@ -1,9 +1,9 @@
 const { NetworkError } = require('@rudderstack/integrations-lib');
-const myAxios = require('../../../util/myAxios');
 const { ENDPOINTS } = require('./config');
 const tags = require('../../util/tags');
 const { getDynamicErrorType } = require('../../../adapters/utils/networkUtils');
 const { JSON_MIME_TYPE } = require('../../util/constant');
+const { httpGET, httpPOST, httpPUT } = require('../../../adapters/network');
```

Comment:
Let's use `handleHttpRequest` utility method instead of `httpGET`, `httpPOST` and `httpPUT` ?

Reply:
Addressed